tests/index.js

```diff
@@ -26,7 +26,7 @@ describe('broccoli-gzip', function(){
     });
 
     builder = new broccoli.Builder(tree);
-    return builder.build().then(function(build) {
+    builder.build().then(function(build) {
```

Comment:
You're correct. Realizing now that Broccoli-builder's build() method is entirely different and the test's structure will need to be rewritten slightly.